src/modules/market-charts/components/market-outcome-charts/market-outcome-charts.jsx

```diff
@@ -21,22 +21,29 @@ export default class MarketOutcomeCharts extends Component {
   static propTypes = {
     currentTimeInSeconds: PropTypes.number,
     excludeCandlestick: PropTypes.bool,
-    fixedPrecision: PropTypes.number.isRequired,
     hasOrders: PropTypes.bool.isRequired,
-    hasPriceHistory: PropTypes.bool,
     isMobile: PropTypes.bool.isRequired,
     marketDepth: PropTypes.object.isRequired,
-    marketId: PropTypes.string,
     maxPrice: PropTypes.instanceOf(BigNumber).isRequired,
     minPrice: PropTypes.instanceOf(BigNumber).isRequired,
     orderBook: PropTypes.object.isRequired,
     orderBookKeys: PropTypes.object.isRequired,
-    outcomeName: PropTypes.string,
-    priceTimeSeries: PropTypes.array,
     selectedOutcome: PropTypes.object.isRequired,
-    updatePrecision: PropTypes.func,
     updateSelectedOrderProperties: PropTypes.func.isRequired,
-    pricePrecision: PropTypes.number.isRequired
+    pricePrecision: PropTypes.number.isRequired,
+    marketId: PropTypes.string.isRequired,
```

Comment:
marketId property is undefined when creating marketOutcomeCharts from create-market-from-liquidity-charts.js container so it can't be required.